### DISCUSSION TITLE
Support snippets like vscode-go

### DISCUSSION THREAD

**Discussion by jkellogg01** [zed-industries/zed#23919] [Feature Requests]
specifically working in go, I'm accustomed to gopls providing snippets in other editors such as `fmain` or `wr` which speeds up development a lot. Zed appears to either not be able to do this, or lack documentation on how to enable it. If the issue is the latter I would be very happy to contribute more detailed documentation surrounding LSP configuration, since there doesn't seem to be a very in-depth documentation source for this topic.

**Comment by mrnugget** [comment]
These are not LSP snippets, but snippets coming from vscode-go: https://github.com/golang/vscode-go/blob/9330b086651a8969c6a8b7a508cf3d8c37f1aae1/extension/snippets/go.json#L173-L177

**Comment by probably-neb** [comment]
Hey! Thanks for bringing this to our attention. You're definitely not the only one who wants this kind of functionality in Zed. We've decided to start moving feature/enhancement requests to discussions so I'm going to move this there.
